cmd/repo.go

```diff
@@ -578,20 +584,13 @@ func (r *RepositoryFile) getRepositories(log *LoggingConfig) (IndexOutputFormat,
 	return indexOutput, nil
 }
 
-func (r *RepositoryFile) getRepository(log *LoggingConfig, repoName string) (IndexOutputFormat, error) {
+func (r *RepositoryFile) getRepository(config *RootCommandConfig, index *IndexYaml, repoName string) (IndexOutputFormat, error) {
```

Comment:
Does the config still need to be passed in here?

Reply:
Good catch, will update now